src/main/java/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval.java

```diff
@@ -334,6 +358,10 @@ private static String hashClasspathEntry(URL entry) throws IOException {
                 return Util.toHexString(digest.digest());
             } catch (NoSuchAlgorithmException x) {
                 throw new AssertionError(x);
+            } finally {
+                if (input != null) {
+                    input.close();;
```

Comment:
Uhm, entry.openStream may open a FileInputStream. I think that the FindBugs complain was valid